### PR TITLE
fix(suite): Fix buy/sell/dca buttons

### PR DESCRIPTION
**Pull request by jvaclavik**
## Related Issue

Resolve 

## Screenshots:

before
image

after
image

### CODE REVIEW

**Comment by seibei-iguchi** [comment]
I don't think this is a good idea. Select bar is a form component, not a navigation one. I'd just create the Subtabs component. Or use Tabs here, since it's supposed to be there anyway.

File: packages/components/src/components/form/SelectBar/SelectBar.tsx [hunk 1]
```diff
@@ -156,6 +163,7 @@ type ValueTypes = number | string | boolean;
 type Option<V extends ValueTypes> = {
     label: ReactNode;
     value: V;
+    icon?: IconName;
```
Comment (seibei-iguchi) [inline packages/components/src/components/form/SelectBar/SelectBar.tsx:166]: iconName

File: packages/components/src/components/form/SelectBar/SelectBar.tsx [hunk 2]
```diff
@@ -22,10 +22,15 @@ import {
     withFrameProps,
 } from '../../../utils/frameProps';
 import { TransientProps } from '../../../utils/transientProps';
+import { Icon, IconName } from '../../Icon/Icon';
+import { Row } from '../../Flex/Flex';
 
 export const allowedSelectBarFrameProps = ['margin', 'width'] as const satisfies FramePropsKeys[];
 type AllowedFrameProps = Pick<FrameProps, (typeof allowedSelectBarFrameProps)[number]>;
 
+export const fillTypes = ['none', 'default'] as const;
```
Comment (seibei-iguchi) [inline packages/components/src/components/form/SelectBar/SelectBar.tsx:31]: Perhaps we should create a common fill types like for other props, it's used in many components.

File: packages/components/src/components/form/SelectBar/SelectBar.tsx [hunk 3]
```diff
@@ -63,14 +68,16 @@ const Options = styled.div<{
...
+    $fillType: FillType;
 }>`
     position: relative;
     display: grid;
     grid-auto-columns: ${({ $optionsCount }) => `minmax(${getPuckWidth($optionsCount)}, 1fr)`};
     grid-auto-flow: column;
     gap: ${spacingsPx.xxs};
     padding: ${spacingsPx.xxs};
-    background: ${mapElevationToBackground};
+    background: ${({ $fillType, theme, $elevation }) =>
```
Comment (seibei-iguchi) [inline packages/components/src/components/form/SelectBar/SelectBar.tsx:79]: Do it the other way around, unset the background when fillType is none. Then you don't have to pass all the props to the mapElevationToBackground.

**Review by seibei-iguchi — CHANGES_REQUESTED** [review]

**Comment by jvaclavik** [comment]
Already fixed in other PR

**Pull request closed without merging**